File: StudentCrudProject/wwwroot/js/Student/Student.Model.js
/// <reference path="../knockout.js" />

var StudentModel = function (item) {
    var self = this;
    item = item || {};
    self.id = ko.observable(item.id || 0)
    self.firstName = ko.observable(item.firstName || '');
    self.lastName = ko.observable(item.lastName || '');
    self.faculty = ko.observable(item.faculty || '');
    self.dob = ko.observable(item.dob || '');
    self.email = ko.observable(item.email || '');
    self.gender = ko.observable(item.gender || '');
}

File: StudentCrudProject/Models/StudentModel.cs
using System.ComponentModel.DataAnnotations;

namespace StudentCrudProject.Models
{
    public class StudentModel
    {
        [Key]
        public int Id { get; set; }

        [Display(Name ="First Name")]
        [StringLength(50)]
        public string FirstName { get; set; }

        [Display (Name ="Last Name")]
        [StringLength(50)]
        public string  LastName {  get; set; }

        [StringLength(50)]
        public string Faculty {  get; set; }

        [Display(Name ="Date Of Birth")]
        public DateTime DOB {  get; set; }
        [EmailAddress]
        [StringLength(50)]
        public string Email {  get; set; }

        [StringLength(10)]
        public string Gender { get; set; }
    }
}
